incubator/mssql-linux/README.md

```diff
@@ -0,0 +1,126 @@
+# HELM Chart for Microsoft SQL Server 2017 on Linux
+
+## Prerequisites
+ * This chart requires Docker Engine 1.8+ in any of their supported platforms.
+ * At least 2GB of RAM (3.25 GB prior to 2017-CU2). Make sure to assign enough memory to the Docker VM if you're running on Docker for Mac or Windows.
+ * Requires the following environment flags
+   - You must change the ACCEPT_EULA in the values.yaml file to `Y` or include `--set ACCEPT_EULA.value=Y` in the command line of `helm install`
+   - Update the template/secret.yaml with your Base64 SA Password
+   - MSSQL_PID=<your_product_id | edition_name> (default: Express)
+   - A strong system administrator (SA) password: At least 8 characters including uppercase, lowercase letters, base-10 digits and/or non-alphanumeric symbols.
+
+## Chart Components
+ * Creates a SQL Server 2017 deployment (default edition: Express)
+ * Creates a Kubernetes Service on specified port (default: 1433)
+ * Creates a Secert to hold SA_PASSWORD
+
+## Installing the Chart
+
+### Creating Secrets for SA Password
+It is important to change the BASE64 text in the templates/secret.yaml file to your own password.  By default, the password is `MySaPassword123`.  The instructions below are how to change the password via terminal window.
+1.  Open a terminal session/window
+2.  Type in the following
+```console
+$ echo -n "<MyNewStrongSaPassword>"| base64
+$ <base 64 output>
+```
+3.  Copy the Base64 output in step 2 into the templates/secret.yaml in the `password` field
```

Comment:
We can do this in the templating similar to https://github.com/kubernetes/charts/blob/master/stable/mysql/templates/secrets.yaml

Reply:
This was a great idea.  I added this to the commit.